crates/bevy_text/src/pipeline.rs

```diff
@@ -209,6 +210,18 @@ impl TextPipeline {
                     .map(move |layout_glyph| (layout_glyph, run.line_y))
             })
             .try_for_each(|(layout_glyph, line_y)| {
+                let mut layout_glyph = layout_glyph.clone();
```

Comment:
Do you expect the compiler to elide this clone when not needed? Otherwise I would move the new variable outside the loop and branch on `font_smoothing` to either update the outer variable and get a reference, or use the `layout_glyph` reference from the iterator.

Reply:
Wasn't able to get the temp variable outside the loop, because it couldn't be left initialized across the closure boundary, and the `LayoutGlyph` type doesn't have a `Default` implementation, so instead I create it inside the closure and conditionally shadow the `layout_glyph` with a reference to it (or with its original reference), which should have the same effect of preventing the extra clone.